### ISSUE TITLE
after update to 16.03 does not work

### ISSUE THREAD

**Issue by brokeniceinteractive** [processone/ejabberd-contrib#161]
i have updated from 16.01 to 16.03, and mod_rest does not work! i have installed module with this line of code:
> ejabberdctl module_install mod_rest

and now it seems that my configuration in ejabberd.yml is ignored:
```
 - 
   port: 5285
   module: ejabberd_http
   request_handlers:
    "/rest": mod_rest

modules
  mod_rest: 
    allowed_ips: 
     - 127.0.0.1 
   access_commands:
     configure:
       - registered_users
       - connected_users
       - register
       - unregister
       - check_account
   allowed_destinations: all
   allowed_stanza_types: all
```
than reading on this post [https://github.com/processone/ejabberd-contrib/issues/159](url) i have moved the conf from ejabberd.yml to $EJABBERDHOME/.ejabberd-source/mod_rest/conf/mod_rest.yml but the problem is this, is i call:
> wget http://localhost:5285/rest --server-response

i obtain:
> Try POSTing a stanza.

but in any other case, for example:
>  wget http://localhost:5285/rest --server-response --post-data 'registered_users locahost'

i obtain this:
```
--2016-04-11 08:29:21--  http://localhost:5285/rest
Connecting to localhost (localhost)|52.73.55.15|:5285... connected.
HTTP request sent, awaiting response... 
  HTTP/1.1 406 Not Acceptable
  Content-Type: text/html; charset=utf-8
  Content-Length: 43
2016-04-11 08:29:21 ERROR 406: Not Acceptable.
```
Any ideas?

**Comment by brokeniceinteractive** [comment]
i have rewrite all configurations, now it seems to work, but i obtain:

Error: account_unprivileged

Is there an authorization?

**Comment by brokeniceinteractive** [comment]
Ok, i i send auth data in post request it works!

wget http://localhost:5285/rest --server-response --post-data '--auth admin localhost password registered_users locahost' i obtain this:

Is this right? in older version i didn't pass --auth, is this secure?

**Comment by badlop** [comment]
> Is this right? in older version i didn't pass --auth, 

If you configure the option access_commands in mod_rest, then it requires auth. If it didn't request it in the past, then it was a bug.

BTW, I fixed a problem in recent ejabberd that broke calling commands!

> is this secure?

Providing the auth credentials of an admin may not be secure enough, so you better define a specific Access, set it in the option, and use its credentials when calling mod_rest.

I've updated the example configuration in README.txt
